app/views/govuk_publishing_components/components/_search.html.erb

```diff
@@ -62,20 +62,22 @@
     <%= tag_label %>
   <% end %>
   <div class="gem-c-search__item-wrapper">
-    <%= tag.input(
-      aria: {
-        controls: aria_controls,
-      },
-      enterkeyhint: "search",
-      class: "gem-c-search__item gem-c-search__input js-class-toggle",
-      id: id,
-      name: name,
-      title: t("components.search_box.input_title"),
-      type: "search",
-      value: value,
-      autocorrect: correction_value,
-      autocapitalize: correction_value,
-    ) %>
+    <div class="gem-c-search__input-wrapper">
```

Comment:
Annoyingly this _is_ necessary 😞 (although I've renamed it to `.js-...` according to your suggestion.

It's because the `accessible-autocomplete` library adds in a new input to the parent container specified in its `element` option (rather than replacing an existing one), so if we give it the parent of the element it'll insert the new field _after_ the submit button, which would require even nastier shenanigans to move around again.

Reply:
Cool, no worries, thanks for the explanation 👍